types/browan/TBSound/meta.json

```diff
@@ -1,21 +1,20 @@
 {
-  "name": "Sound Level Sensor, TBSL100, Tabs",
+  "name": "Tabs Sound Level Sensor, TBSL100",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed